lib/data/external/GCP/GcpApis/getTagging.js

```diff
@@ -0,0 +1,26 @@
+const async = require('async');
+
+const { retrieveTags } = require('../GcpUtils');
+
+function getObjectTagging(params, callback) {
+    return async.waterfall([
```

Comment:
Can avoid using the waterfall since there is just one asynchronous method here.

Reply:
Will fix the extra steps